.github/labeler.yml

```diff
@@ -1,26 +1,31 @@
 ---
 ADR:
-- adr/*.md
+  - changed-files:
+    - adr/*.md
 
 documentation:
-- README.md
-- data/*.md
-- cms/*.md
+  - changed-files:
+    - README.md
+    - data/*.md
+    - cms/*.md
 
 terraform:
-- terraform/**/*
-- terraform-azure/**/*
+  - changed-files:
+    - terraform-azure/**/*
```

Comment:
Does `terraform/**/*` need to be included here as well?

Reply:
No that folder contained GPaaS only code removed recently in this PR
https://github.com/DFE-Digital/early-years-foundation-recovery/pull/991